tests/mocha/astnode_test.js

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+goog.module('ASTTest');
+
+const {sharedTestSetup, sharedTestTeardown} = goog.require('TestHelpers');
+
```

Comment:
I believe that @moniika has been promulgating a standard of putting two blank lines between the imports (or module declaration, if there are no imports) and the main body of the file.
```suggestion


```